doc/akri_connector/connector-metadata-schema.json

```diff
@@ -0,0 +1,305 @@
+          "required": [
+            "containerRegistryName",
+            "imageName",
+            "digest"
+          ]
+        }
+      ]
+    },
+    "helmConfigurationSettings": {
+      "type": "object",
+      "properties": {
+        "containerRegistryName": {
+          "type": "string",
+          "description": "The container registry name that holds the helm chart for the connector. For example: contoso-acr.io"
+        },
+        "helmChartName": {
+          "type": "string",
+          "description": "The helm chart name for the connector. For example: helm/MyConnector"
+        },
+        "tag": {
+          "type": "string",
+          "description": "The tag of the helm chart for this connector"
+        },
+        "digest": {
+          "type": "string",
+          "description": "The digest of the helm chart for this connector"
+        }
+      },
+      "oneOf": [
+        {
+          "required": [
+            "containerRegistryName",
+            "helmChartName",
+            "tag"
+          ]
+        },
+        {
+          "required": [
+            "containerRegistryName",
+            "helmChartName",
+            "digest"
+          ]
+        }
+      ]
+    },
+    "kubernetes": {
+      "type": "object",
+      "description": "The kubernetes-level metadata",
+      "properties": {
+        "minimumKubernetesVersion": {
+          "type": "string",
+          "description": "Minimum required version of kubernetes needed to run this connector"
+        }
+      }
+    },
+    "secrets": {
+      "type": "object",
+      "description": " Secrets that will be mounted onto all connector instances.",
+      "properties": {
+        "secretAlias": {
+          "type": "string",
+          "description": "The application-defined alias for the secret"
+        },
+        "secretKey": {
+          "type": "string",
+          "description": "The key in the secret to be mounted"
+        },
+        "secretRef": {
+          "type": "string",
+          "description": "The name of the secret to be mounted"
+        }
+      },
+      "required": [
+        "secretAlias",
+        "secretKey",
+        "secretRef"
+      ]
+    },
+    "architecture": {
+      "type": "string",
+      "description": "The architecture that this connector should run on"
+    },
+    "sourceCode": {
+      "type": "object",
+      "description": "Details about the source code that compiled this connector",
+      "properties": {
+        "language": {
+          "type": "string",
+          "description": "The programming language used to write this connector"
+        },
+        "languageVersion": {
+          "type": "string",
+          "description": "The version of the programming language used to write this connector"
+        },
+        "sdk": {
+          "type": "object",
+          "description": "Details about the versions of the AIO SDKs used to write this connector (if they were used)",
+          "properties": {
```

Comment:
For what are these versions used?

If there is no use case, then we should not add them.

Reply:
https://github.com/Azure/iot-operations-sdks/pull/796#discussion_r2109837305